Projects/Foundations/Schemas/Docs-Topics/T/Turtles/Turtles-Data-Mine/turtles-data-mine-001-turtles.json

```diff
@@ -0,0 +1,83 @@
+{
+    "topic": "Turtles Data Mine",
+    "pageNumber": "1",
+    "type": "Turtles",
+    "definition": {
+        "text": "The Turtles Data Mine features the famous System 1 trading indicator as created by Richard Dennis and Bill Eckhardt in the early 1980s."
+    },
+    "paragraphs": [
+        {
+            "style": "Title",
+            "text": "Turtles on the Charts"
+        },
+		{
+            "style": "Text",
+            "text": "This trading system is a trend following system that attempts to catch a breakout. It is similar to a Donchian Breakout strategy."
+        },
+		{
+            "style": "Link",
+            "text": "Turtles System 1(investopedia)->investopedia.com/articles/trading/08/turtle-trading.asp"
```

Comment:
Here the same.

Reply:
Just had a look and this is interesting @gitforkgit . I've been using templates from existing docs from way back and haven't noticed what's going on until now; I use notepad++ and it looks like my default setup is 4 spaces = 1 tab, whereas github here must parse it differently. The template I use has 2 tabs at opening bracket and 8 spaces for the closing one. 
What shall our standard here be?

Reply:
@quantum8 @harrellbm You once commented what the standard is or was at that point. Do you know what the standard is right now?

Reply:
I believe it has been 4 across other places in the system. So let's go with that